### ISSUE TITLE
Problemi con pwm 

### ISSUE THREAD

**Issue by YggdrasilUrthr** [GiorgioChiurato/Linux-Revolution#1]
E' ancora work in progress un sistema di fade da applicare al puntamento manuale. Il più grande ostacolo è rappresentato da un errore nel fade di entrambi i led; infatti solo uno di due led si comporta come dovrebbe mentre l'altro rimane sempre spento.

**Comment by ghost** [comment]
Se ho capito bene, un led deve rimanere acceso quando manuale e l'altro spento; quando è automatico, si invertono, corretto?

**Comment by YggdrasilUrthr** [comment]
No, il meccanismo è molto più complicato. La parte con l'errore non è comunque inclusa nella parte di codice che ho pubblicato. In ogni caso il meccanismo sarebbe che quando il meccanismo è in regolazione manuale (ogni volta che si preme un pulsante l'angolo del servo aumenta e se si preme l'altro diminuisce di un grado) a seconda della distanza dal valore ottenuta dalla misura automatica un led aumenta di intensità. Ci sono due led, uno per indicare la correzione negativa, l'altro per indicare una correzione positiva. Il problema è che un led funzione come dovrebbe mentre l'altro (anche se ha la stessa parte di codice con variabili diverse) non funziona.

Inviato da iPad

**Comment by ghost** [comment]
Mmm... bisognerebbe vedere il codice per capire dove non funziona. Riesci a fornirlo?

**Comment by YggdrasilUrthr** [comment]
Oggi lo carico

Inviato da iPad
